babel.config.js

```diff
@@ -1,11 +1,12 @@
 
 module.exports = function(api){
-    api.cache(true);
```

Comment:
Was equivalent to `api.cache.forever()`, which not meant to be used alongside [api.env](https://babeljs.io/docs/en/config-files#apienv) in its doc note.